{"name": "test_unsuccessful_login[chrome- or 1=1 -- --1212]", "status": "passed", "attachments": [{"name": "log", "source": "b7fc9977-ad97-4470-b1c6-11245d8a070b-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "get_browser", "value": "'chrome'"}, {"name": "username", "value": "' or 1=1 -- -'"}, {"name": "password", "value": "1212"}], "start": 1622460055843, "stop": 1622460056353, "uuid": "1a52df2e-4192-49bb-8a93-d9e73f195241", "historyId": "2c2626598a52618999f833b036198faa", "testCaseId": "1ce5752f9bf5ac0dd16171f6da2bde9d", "fullName": "test_cases.test_user_sign_in.Test_UserSignIn#test_unsuccessful_login", "labels": [{"name": "story", "value": "Cloud Login"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "@pytest.mark.usefixtures('add_logs_on_failure', 'get_browser')"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_user_sign_in"}, {"name": "subSuite", "value": "Test_UserSignIn"}, {"name": "host", "value": "DESKTOP-57RCRK3"}, {"name": "thread", "value": "9200-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_user_sign_in"}]}